src/styles/blocks/product-card.css

```
@@ -0,0 +1,95 @@
+.product-card {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: space-between;
+  max-width: 200px;
+  padding-top: 32px;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.product-card__img {
+  width: 160px;
+  height: 134px;
+  margin-bottom: 40px;
+}
+
+.product-card__content {
+  padding: 0 16px;
+}
+
+.product-card__title {
+  font-size: 12px;
+  font-weight: 500;
+  margin-bottom: 4px;
+  line-height: 18px;
+}
+
+.product-card__code {
+  line-height: 14px;
+  color: var(--secondary);
+  margin-bottom: 16px;
+
+}
+
+.product-card__rate {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  margin-bottom: 24px;
+}
+
+.product-card__review {
+  margin-left: 22px;
+}
+
+.product-card__price-content {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  margin-bottom: 16px;
+}
+
+.product-card__price-title {
+  font-size: 12px;
+  line-height: 18px;
+  color: var(--secondary);
+  margin-right: 85px;
+}
```

It would be better to set `justify-content: space-between` for `product-card__price-content`, rather than setting the property to `center` and than setting `margin-right` for `product-card__price-title`. You can also apply the approach to `product-card__rate`

```suggestion
.product-card__price-content {
  display: flex;
  align-items: center;
  justify-content: space-between;
  margin-bottom: 16px;
}

.product-card__price-title {
  font-size: 12px;
  line-height: 18px;
  color: var(--secondary);
}
```